### PR TITLE
PAR Liudmyla Iefremova 

### PR DESCRIPTION
**Pull request by LiudmilaE**
Celebrities - Done!

### CODE REVIEW

**Comment by LiudmilaE** [comment]
Fixed bug with routers - credit to Elodie WANANG
))))

File: mongooseMovies/bin/seeds.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/mongooseMovies', {useMongoClient: true});
+const Celebrity = require('../models/celebrity');
+
+const celebrities = [
+  {
+    name: 'Michael',
+    occupation: 'Ironhack TA',
+    catchPhrase: 'And now... you should create the LOGIC!',
```
Comment (mlande) [inline mongooseMovies/bin/seeds.js]: Haha :)

File: mongooseMovies/models/celebrity.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+const Schema   = mongoose.Schema;
+
+const celebritySchema = new Schema({
+  name : { type: String, required: [true, 'Please enter the name'] },
```
Comment (mlande) [inline mongooseMovies/models/celebrity.js]: Nice use of mongoose Schema

File: mongooseMovies/views/celebrities/index.ejs [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+<ul>
+	<% celebrities.forEach((celebrity) => { %>
+		<li>
+			<a href="/celebrities/<%= celebrity.id %>"><%= celebrity.name %></a>
+		</li>
+
+		<form action="/celebrities/<%= celebrity.id %>/delete" method="POST">
+			<button>Delete celebrity</button>
+		</form>
+	<% }); %>
```
Comment (mlande) [inline mongooseMovies/views/celebrities/index.ejs]: You just forgot to add a link to edit the celeb

**Comment by mlande** [comment]
Very nicely donem Liuda.
The only thing you forgot is to give the user the ability to access the edit page, even though you already created the http handler for it and the view. Always remember what they user needs to see or do :)

Keep up the good work

**Pull request closed without merging**